### PR TITLE
Add total hits param

### PR DESCRIPTION
**Pull request by phieronymus**
Linked to https://univapay-systems.slack.com/archives/CA0U3CSQ1/p1683822852597689

### CODE REVIEW

File: src/resources/CRUDResource.ts [hunk 1]
```diff
@@ -12,17 +12,22 @@ export enum CursorDirection {
...
-export interface CRUDItemsResponse<A> {
+export type CRUDItemsResponse<A> = {
     items: A[];
     hasMore: boolean;
-}
+    totalHits?: number;
+};
+
+export type CRUDAOSItemsResponse<A> = CRUDItemsResponse<A> & {
+    totalHits?: number;
```
Comment (gpapako) [inline src/resources/CRUDResource.ts:29]: do we need this? Cause we just added totalHits to CRUDItemsResponse 🤔

Comment (gpapako) [reply src/resources/CRUDResource.ts:29]: Actually.. maybe we should not have added it to CRUDItemsResponse? 🤔

**Review by gpapako — COMMENTED** [review]
Just a comment. We can merge after resolving this

File: src/resources/CRUDResource.ts (hunk 1 above)
Comment (phieronymus) [reply src/resources/CRUDResource.ts:29]: Oh yeah, you are right. I added it to the wrong place and apparently forgot to remove it from there. Removing from `CRUDItemsResponse`.

**Review by gpapako — APPROVED** [review]
LGTM!

**Comment by gpapako** [comment]
Merging!

**Pull request merged**